infra/tf/betteruptime/main.tf

```diff
@@ -0,0 +1,48 @@
+provider "betteruptime" {
+	api_token = module.betteruptime_secrets.values["betteruptime/token"]
+}
+
+module "betteruptime_secrets" {
+	source = "../modules/secrets"
+	keys = [
+		"betteruptime/token",
+		"rivet/api_status/token",
+	]
+}
+
+resource "betteruptime_status_page" "status_page" {
```

Comment:
Do you think it'd make sense to add the better uptime status page domain as a TF output just for clarify?

Reply:
Resolved in https://github.com/rivet-gg/rivet/commit/0bd6e1adc4f518e2e88f822971369caf047bf553

I think this makes sense, but will it get swept away with the rest of the logs from Bolt? I'm not too familiar with TF outputs.

Reply:
Yeah. Idk, it's nice if you know what you're doing but most people won't.